docs/blog/pycon-2024-great-tables-are-possible/index.qmd

```diff
@@ -79,9 +79,10 @@ We’ll cover the following:
 
 Check out these resources to learn more about the wild and beautiful life of display tables:
 
-* [Great Tables example gallery](/docs/examples)
```

Comment:
shoot some of these broken links were my bad -- but it looks like the fixes may also be broken 😓.